diff --git a/android/Android.mk b/android/Android.mk
index bc82d1a..3e12cbc 100644
--- a/android/Android.mk
+++ b/android/Android.mk
@@ -50,6 +50,8 @@ MESA_VK_LIB_SUFFIX_freedreno := freedreno
 MESA_VK_LIB_SUFFIX_broadcom := broadcom
 MESA_VK_LIB_SUFFIX_panfrost := panfrost
 MESA_VK_LIB_SUFFIX_virtio := virtio
+MESA_VK_LIB_SUFFIX_asahi := asahi
+MESA_VK_LIB_SUFFIX_imagination := imagination
 MESA_VK_LIB_SUFFIX_swrast := lvp_mesa3d
 
 include $(CLEAR_VARS)
diff --git a/android/mesa3d_cross.mk b/android/mesa3d_cross.mk
index c3dec2e..077ea7b 100644
--- a/android/mesa3d_cross.mk
+++ b/android/mesa3d_cross.mk
@@ -259,7 +259,7 @@ $(MESON_GEN_FILES_TARGET): $(sort $(shell find -L $(MESA3D_TOP) -not -path '*/\.
 		"cpp = [$(foreach arg,$(PRIVATE_CXX),'$(subst prebuilts/,$(AOSP_ABSOLUTE_PATH)/prebuilts/,$(arg))',)'']\n"        \
 		"c_ld = 'lld'\n"                                                                                                  \
 		"cpp_ld = 'lld'\n\n"                                                                                              \
-		"pkgconfig = ['env', 'PKG_CONFIG_LIBDIR=' + '$(call relative-to-absolute,$(MESON_GEN_DIR))', '/usr/bin/pkg-config']\n\n" \
+		"pkg-config = ['env', 'PKG_CONFIG_LIBDIR=' + '$(call relative-to-absolute,$(MESON_GEN_DIR))', '/usr/bin/pkg-config']\n\n" \
 		"llvm-config = '/dev/null'\n"                                                                                     \
 		"[host_machine]\n"                                                                                                \
 		"system = 'android'\n"                                                                                              \
